bowtie/_cli.py

```diff
@@ -471,7 +471,7 @@ def sequenced(cases, reporter):
 
 def suite_cases_from(files, remotes):
     for file in files:
-        if file.name == "refRemote.json" or file.name == "dynamicRef.json":
+        if file.name == "refRemote.json" or file.name == "dynamicRef.json" or file.name == "vocabulary.json":
```

Comment:
:/ -- it now probably makes sense to do this as:

```suggestion
        if file.stem in {"refRemote", "dynamicRef", "vocabulary"}:
```

Reply:
I guess it should be array literal not map literal

Reply:
That's a set literal (with no colons).

Reply:
(CI is failing because the line is now too long -- `.stem` will trim off the `.json`, which probably fits).